tests/test_app.py

```diff
@@ -1,14 +1,28 @@
+import mock
 import unittest
+
 from ubuntucleaner.main import UbuntuCleanerWindow
 
 
-class TestApp(unittest.TestCase):
+class TestUbuntuCleanerWindow(unittest.TestCase):
     def setUp(self):
         self.window = UbuntuCleanerWindow()
 
+    def tearDown(self):
+        del self.window
+
     def test_app(self):
         self.assertEqual(self.window.feature_dict, {'janitor': 0})
         self.assertIsNotNone(self.window.aboutdialog)
 
-    def tearDown(self):
-        del self.window
+    def test_on_about_button_clicked(self):
+        with mock.patch("ubuntucleaner.main.Gtk.AboutDialog.run") as m_run:
+            self.window.about_button.clicked()
```

Comment:
Simulates the `clicked()` event

Reply:
Yes I agree ideally it should what the GUI is actually doing, but since it's blocking it probably needs to be done through a thread and all of that, which for now is harder to implement that just mocking few calls